app/controllers/contestant_projects_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ContestantProjectsController < ApplicationController
+  def create
+    project = Project.find(params[:project].to_i)
+    contestant = Contestant.find(params[:contestant].to_i)
+    ContestantProject.create(contestant: contestant, project: project)
```

Comment:
Instead of grabbing the project and contestant again, you can possibly use the available params in your form. 

`ContestantProject.create(contestant_id: params[:contestant_id], project_id: params[:project_id])`